packages/foundation/jest.config.js

```diff
@@ -0,0 +1,14 @@
+let { createConfig } = require('../../scripts/tasks/jest-resources');
+let path = require('path');
+
+const config = createConfig({
+  setupFiles: [path.resolve(path.join(__dirname, 'config', 'tests.js'))],
+
+  moduleNameMapper: {
+    'office-ui-fabric-react/lib/': 'office-ui-fabric-react/lib-commonjs/'
```

Comment:
I don't think we need this for foundation

Reply:
True, tests shouldn't be using OUFR.